### PR TITLE
Updated string to reflect beta.4 version

### PR DESCRIPTION
**Pull request by adityapatwardhan**
Updated strings in md, docker and yml and sh files to reflect the beta.4 version.

### CODE REVIEW

File: docker/release/nanoserver-insider/Dockerfile [hunk 1]
```diff
@@ -10,7 +10,7 @@ ARG NanoServerRepo=microsoft/nanoserver-insider
 FROM ${WindowsServerCoreRepo}:$WindowsServerCoreVersion  AS installer-env
 
 # Arguments for installing powershell, must be defined in the container they are used
-ARG PS_VERSION=6.0.0-beta.3
+ARG PS_VERSION=6.0.0-beta.4
 ARG PS_DOWNLOAD_SHA=FF45FE7D184B50297CA706EFAEA4E37C59807D33577521A715C147BCD5098B01
```
Comment (daxian-dbw) [inline docker/release/nanoserver-insider/Dockerfile]: This hash value should be changed to
`DAB8CC377D1BADD91688AEC9277D7696CC8DB2C25CECBCC934BD91AA7C724901`
This is the SHA256 hash value of `PowerShell-6.0.0-beta.4-win10-win2016-x64.zip`

File: docs/installation/linux.md [hunk 2]
```diff
@@ -443,12 +443,12 @@ chmod a+x PowerShell-6.0.0-alpha.18-x86_64.AppImage
 ### Uninstallation - Kali
```
Comment (daxian-dbw) [inline docs/installation/linux.md]: `alpha.18` is used in the `Kali` section, and they need to be updated to `beta.4`.

**Comment by daxian-dbw** [comment]
Travis CI is expected to fail because it depends on `tools/download.sh`. However, I don't see you update the `.spelling` file. Did you do the spellchecking? See https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#spellchecking-documentation

File: docker/release/nanoserver-insider/Dockerfile (hunk 1 above)
Comment (adityapatwardhan) [inline docker/release/nanoserver-insider/Dockerfile]: Fixed.

File: docs/installation/linux.md (hunk 2 above)
Comment (adityapatwardhan) [inline docs/installation/linux.md]: Fixed.

**Comment by adityapatwardhan** [comment]
I was surprised as well that I need not update the .spellings file. Maybe I need to update with the CHANGELOG.md.

PS D:\OpenPowershell\aditya\PowerShell> git status
On branch beta4.updatestrings
nothing to commit, working tree clean
PS D:\OpenPowershell\aditya\PowerShell> mdspell "**/*.md" --ignore-numbers --ignore-acronyms
Spelling - CHANGELOG.md
Spelling - CODE_OF_CONDUCT.md
Spelling - demos/Apache/readme.md
Spelling - demos/Azure/README.md
Spelling - demos/crontab/README.md
Spelling - demos/DSC/readme.md
Spelling - demos/install/README.md
Spelling - demos/powershellget/README.md
Spelling - demos/python/README.md
Spelling - demos/README.md
Spelling - demos/rest/README.md
Spelling - demos/SSHRemoting/README.md
Spelling - demos/SystemD/readme.md
Spelling - docker/README.md
Spelling - docs/building/internals.md
Spelling - docs/building/linux.md
Spelling - docs/building/macos.md
Spelling - docs/building/windows-core.md
Spelling - docs/building/windows-full.md
Spelling - docs/community/governance.md
Spelling - docs/debugging/README.md
Spelling - docs/dev-process/breaking-change-contract.md
Spelling - docs/dev-process/coding-guidelines.md
Spelling - docs/dev-process/map-json-files.md
Spelling - docs/dev-process/resx-files.md
Spelling - docs/FAQ.md
Spelling - docs/git/basics.md
Spelling - docs/git/README.md
Spelling - docs/git/submodules.md
Spelling - docs/host-powershell/README.md
Spelling - docs/installation/linux.md
Spelling - docs/installation/windows.md
Spelling - docs/KNOWNISSUES.md
Spelling - docs/learning-powershell/create-powershell-scripts.md
Spelling - docs/learning-powershell/debugging-from-commandline.md
Spelling - docs/learning-powershell/powershell-beginners-guide.md
Spelling - docs/learning-powershell/README.md
Spelling - docs/learning-powershell/using-vscode.md
Spelling - docs/learning-powershell/working-with-powershell-objects.md
Spelling - docs/maintainers/issue-management.md
Spelling - docs/maintainers/pull-request-process.md
Spelling - docs/maintainers/README.md
Spelling - docs/maintainers/releasing.md
Spelling - docs/testing-guidelines/CodeCoverageAnalysis.md
Spelling - docs/testing-guidelines/PowerShellCoreTestStatus.md
Spelling - docs/testing-guidelines/testing-guidelines.md
Spelling - docs/testing-guidelines/TestRoadmap.md
Spelling - docs/testing-guidelines/WritingPesterTests.md
Spelling - README.md
Spelling - src/libpsl-native/README.md
Spelling - src/Microsoft.PowerShell.PSReadLine/en-US/PSReadline.md
Spelling - src/Microsoft.PowerShell.SDK/README.md
Spelling - src/Modules/README.md
Spelling - src/Modules/Shared/Pester/CHANGELOG.md
Spelling - src/Modules/Shared/Pester/README.md
Spelling - src/powershell-win-core/bin/codecoverage/netcoreapp2.0/win10-x64/Modules/Pester/CHANGELOG.md
Spelling - src/powershell-win-core/bin/codecoverage/netcoreapp2.0/win10-x64/Modules/Pester/README.md
Spelling - src/powershell-win-core/bin/codecoverage/netcoreapp2.0/win10-x64/publish/Modules/Pester/CHANGELOG.md
Spelling - src/powershell-win-core/bin/codecoverage/netcoreapp2.0/win10-x64/publish/Modules/Pester/README.md
Spelling - src/powershell-win-core/bin/Debug/netcoreapp2.0/win10-x64/Modules/Pester/CHANGELOG.md
Spelling - src/powershell-win-core/bin/Debug/netcoreapp2.0/win10-x64/Modules/Pester/README.md
Spelling - src/powershell-win-core/bin/Debug/netcoreapp2.0/win10-x64/publish/Modules/Pester/CHANGELOG.md
Spelling - src/powershell-win-core/bin/Debug/netcoreapp2.0/win10-x64/publish/Modules/Pester/README.md
Spelling - src/powershell/README.md
Spelling - src/TypeCatalogGen/README.md
Spelling - test/csharp/README.md
Spelling - test/powershell/README.md
Spelling - test/README.md
Spelling - test/tools/CodeCoverageAutomation/README.md
>> 69 files are free from spelling errors

File: docs/installation/linux.md [hunk 3]
```diff
@@ -431,24 +431,24 @@ powershell
 
 ```sh
 # Grab the latest App Image
-wget https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-alpha.18/PowerShell-6.0.0-alpha.18-x86_64.AppImage
+wget https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-alpha.18/PowerShell-6.0.0-beta.4-x86_64.AppImage
```
Comment (daxian-dbw) [inline docs/installation/linux.md]: > **download/v6.0.0-alpha.18**

This needs to be changed too.

Comment (adityapatwardhan) [inline docs/installation/linux.md]: Fixed. Sorry missed that.

**Review by daxian-dbw — APPROVED** [review]

**Pull request merged**